sources/packages/backend/libs/integrations/src/services/ssh/sftp-integration-base.ts

```diff
@@ -66,6 +66,112 @@ export abstract class SFTPIntegrationBase<DownloadType> {
     }
   }
 
+  /**
+   * Converts input into an ASCII 7 bit buffer.
+   * @param rawContent Raw content in string format.
+   * @returns a buffer of the input with extended ASCII accented characters converted to unaccented characters.
+   */
+  convertRawContent(rawContent: string) {
+    const content = Buffer.from(rawContent, FILE_DEFAULT_ENCODING);
+    for (const [index, char] of content.entries()) {
+      if (char > 127) {
+        // if extended ascii
+        switch (char) {
+          case 192:
+          case 193:
+          case 194:
+          case 195:
+          case 196:
+          case 197: // ÀÁÂÃÄÅ
+            content[index] = 65; // replace with A
+            break;
+          case 199: // Ç
+            content[index] = 67; // replace with C
+            break;
+          case 200:
+          case 201:
+          case 202:
+          case 203: // ÈÉÊË
+            content[index] = 69; // replace with E
+            break;
+          case 204:
+          case 205:
+          case 206:
+          case 207: // ÌÍÎÏ
+            content[index] = 73; // replace with I
+            break;
+          case 209: // Ñ
+            content[index] = 78; // replace with N
+            break;
+          case 210:
+          case 211:
+          case 212:
+          case 213:
+          case 214: // ÒÓÔÕÖ
+            content[index] = 79; // replace with O
+            break;
+          case 217:
+          case 218:
+          case 219:
+          case 220: // ÙÚÛÜ
+            content[index] = 85; // replace with U
+            break;
+          case 223: // ß
+            content[index] = 115; // replace with s
```

Comment:
I would recommend letting this one be converted to "?" unless there is a reason to keep it.

Reply:
updated